qlib/mod.rs

```diff
@@ -1186,14 +1192,20 @@ impl CompleteEntry {
 }
 
 pub struct UringQueue {
+    #[cfg(not(feature = "cc"))]
     pub submitq: QMutex<VecDeque<UringEntry>>,
+    #[cfg(feature = "cc")]
+    pub submitq: ArrayQueue<UringEntry>,
```

Comment:
why not Mutex in CC? And why ArrayQueue instead of VecDeque?

Reply:
1.ArrayQueue is from Crossbeam crate, which is designed for lock free concurrency.
2.ArrayQueue instead of VecDeque: avoid dynamic growth, arrayqueue will return an error when the queue is full.